### PR TITLE
Web socket test - StubMethodsTest

### PR DESCRIPTION
**Pull request by theJNOVAK**

### CODE REVIEW

File: tests/org.jboss.tools.ws.ui.bot.test/src/org/jboss/tools/ws/ui/bot/test/SocketWSBotTests.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package org.jboss.tools.ws.ui.bot.test;
```
Comment (psrna) [inline tests/org.jboss.tools.ws.ui.bot.test/src/org/jboss/tools/ws/ui/bot/test/SocketWSBotTests.java]: Please add copyright info

File: tests/org.jboss.tools.ws.ui.bot.test/src/org/jboss/tools/ws/ui/bot/test/websocket/StubMethodsHelper.java [hunk 2]
```diff
@@ -0,0 +1,210 @@
+package org.jboss.tools.ws.ui.bot.test.websocket;
```
Comment (psrna) [inline tests/org.jboss.tools.ws.ui.bot.test/src/org/jboss/tools/ws/ui/bot/test/websocket/StubMethodsHelper.java]: please add copyright info

**Comment by psrna** [comment]
Please squash the commits into one with reasonable info

**Comment by psrna** [comment]
Please remove unused imports.

File: tests/org.jboss.tools.ws.ui.bot.test/src/org/jboss/tools/ws/ui/bot/test/SocketWSBotTests.java (hunk 1 above)
Comment (theJNOVAK) [inline tests/org.jboss.tools.ws.ui.bot.test/src/org/jboss/tools/ws/ui/bot/test/SocketWSBotTests.java]: Added

File: tests/org.jboss.tools.ws.ui.bot.test/src/org/jboss/tools/ws/ui/bot/test/websocket/StubMethodsHelper.java (hunk 2 above)
Comment (theJNOVAK) [inline tests/org.jboss.tools.ws.ui.bot.test/src/org/jboss/tools/ws/ui/bot/test/websocket/StubMethodsHelper.java]: Added

File: tests/org.jboss.tools.ws.ui.bot.test/src/org/jboss/tools/ws/ui/bot/test/websocket/StubMethodsTest.java [hunk 3]
```diff
@@ -0,0 +1,113 @@
+package org.jboss.tools.ws.ui.bot.test.websocket;
```
Comment (theJNOVAK) [inline tests/org.jboss.tools.ws.ui.bot.test/src/org/jboss/tools/ws/ui/bot/test/websocket/StubMethodsTest.java]: Added

**Comment by jkopriva** [comment]
testPR

**Comment by theJNOVAK** [comment]
Updated.
@psrna or @rhopp or @rawagner please review before merge.

**Pull request merged**